en/advanced/best-practice-security.md

```diff
@@ -152,6 +153,16 @@ app.use(session({
 }))
 ```
 
+## Prevent brute-force attacks against authorisation
+
+Make sure login endpoints are protected to make private data more secure.
+
+One simple and in the same time quite powerful way is to block authorisation attempts by 2 metrics:
+1. The first is number of consecutive failed attempts by Username and IP pair. 
+1. The second is number of failed attempts from IP per long period of time. For example, block IP on 100 failed attempts per day.
```

Comment:
...attempts from an IP address over some long period of time. For example, block an IP address if it makes 100 failed attempts in one day.